docs/PacketFence_Clustering_Guide.asciidoc

```diff
@@ -1117,6 +1117,23 @@ Then reload the file and reboot the server:
   # sysctl -p
   # reboot
 
+
+IP addresses in a cluster environment
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+DHCP and DNS services
+^^^^^^^^^^^^^^^^^^^^^
+
+In registration and isolation networks, DHCP and DNS servers are physical IP addresses of *all* cluster members.
```

Comment:
I think this should be rephrased - it's not clear to me.

Perhaps:

In registration and isolation networks, the DHCP and DNS services are bound to the physical IP address of each cluster member.